cfgov/unprocessed/js/modules/post-filter.js

```diff
@@ -102,14 +102,30 @@ PostFilter.prototype = {
   },
 
   onSubmit: function( e ) {
-    var dateRange =
-      dateRangeFormatter.format( this.$gte.val(), this.$lte.val() );
-    if ( dateRange && dateRange.isValid ) {
+    var $form = $( e.target );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/606b7c1d369eb210b18ded6cb7f660ffbe3bf87e